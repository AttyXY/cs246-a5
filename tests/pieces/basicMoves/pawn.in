setup
+ k a1
+ K h8
done
game human human
move g2 g4
move a1 a2
move g4 g5
move a2 a1
quit
quit
