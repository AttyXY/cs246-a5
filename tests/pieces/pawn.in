setup
+ k a1
+ K h8
+ p b2
+ P b6
= black
done
game human human
move b2 b4
move a1 a2
move b4 b5
move b5 b6
move b6 b8
move b6 b7
quit
quit
