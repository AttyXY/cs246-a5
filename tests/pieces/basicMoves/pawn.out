WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING SETUP

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  k-------
   abcdefgh

8  -------K
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  k-------
   abcdefgh

8  -------K
7  --------
6  --------
5  --------
4  --------
3  --------
2  ------P-
1  k-------
   abcdefgh

STARTING GAME!

8  -------K
7  --------
6  --------
5  --------
4  --------
3  --------
2  ------P-
1  k-------
   abcdefgh

8  -------K
7  --------
6  --------
5  --------
4  ------P-
3  --------
2  --------
1  k-------
   abcdefgh

8  -------K
7  --------
6  --------
5  --------
4  ------P-
3  --------
2  k-------
1  --------
   abcdefgh

8  -------K
7  --------
6  --------
5  ------P-
4  --------
3  --------
2  k-------
1  --------
   abcdefgh

8  -------K
7  --------
6  --------
5  ------P-
4  --------
3  --------
2  --------
1  k-------
   abcdefgh

Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
